packages/manager/modules/office/src/dashboard/office/license/user/user.module.js

```diff
@@ -2,13 +2,19 @@ import angular from 'angular';
 import 'angular-translate';
 import '@ovh-ux/ui-kit';
 import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import ovhManagerAdvices from '@ovh-ux/manager-advices';
 
 import routing from './user.routing';
 
 const moduleName = 'ovhManagerOfficeLicenseUser';
 
 angular
-  .module(moduleName, [ngTranslateAsyncLoader, 'oui', 'pascalprecht.translate'])
+  .module(moduleName, [
+    ngTranslateAsyncLoader,
+    'oui',
+    'pascalprecht.translate',
+    ovhManagerAdvices,
```

Comment:
same as here:
https://github.com/ovh/manager/pull/4597/files#r607874534

Reply:
Thanks. Its not mandatory to sort module dependencies but its mandatory to sort all imports.